components/service/fretboard/src/main/java/mozilla/components/service/fretboard/scheduler/workmanager/WorkManagerSyncScheduler.kt

```diff
@@ -35,7 +35,7 @@ class WorkManagerSyncScheduler {
         val constraints = Constraints.Builder()
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
-        val syncWork = PeriodicWorkRequest.Builder(SyncWorker::class.java, 1, TimeUnit.DAYS)
+        val syncWork = PeriodicWorkRequest.Builder(worker, 1, TimeUnit.DAYS)
```

Comment:
Maybe we can move the interval to be an optional parameter, like:

```Kotlin
    fun schedule(
        worker: Class<out SyncWorker>,
        interval: Pair<Long, TimeUnit> = Pair(1, TimeUnit.DAYS)
    ) {
```

Reply:
Changed